DemoNativeControls/app/src/main/java/com/jwplayer/demo/nativecontrols/JWPlayerNativeControls.java

```diff
@@ -0,0 +1,253 @@
+            case R.id.mute_button:
+                if(mPlayerView.getState() == PlayerState.PLAYING){
+                    boolean muted = mPlayerView.getMute();
+                    if(muted){ //we're gonna unmute it
+                        mMuteButton.setText("Mute");
+                    } else { //We're gonna mute it
+                        mMuteButton.setText("UnMute");
+                    }
+                    mPlayerView.setMute(!muted);
+                }
+                break;
+        }
+
+        //Notify all our listeners that one of the mNativePlayerControls has been interacted with
+        for (OnControlsInteraction ocl: mControlChangeListeners) {
+            ocl.OnControlsInteractedWith(stoppedPlayback);
+        }
+    }
+
+    @Override
+    public void onPlay(PlayEvent playEvent) {
+        mPlayPauseButton.setImageResource(R.drawable.exo_controls_pause);
+    }
+
+    @Override
+    public void onPause(PauseEvent pauseEvent) {
+        mPlayPauseButton.setImageResource(R.drawable.exo_controls_play);
+    }
+
+    @Override
+    public void onComplete(CompleteEvent completeEvent) {
+        mPlayPauseButton.setImageResource(R.drawable.exo_controls_play);
+    }
+
+    private class JWProgressBar implements VideoPlayerEvents.OnTimeListener,
+            SeekBar.OnSeekBarChangeListener {
+        //Ignore time events when the user is using the scrub to navigate the video otherwise the scrubber will jump erratically
+        private boolean ignoreTimeEvents = false;
+
+        @Override
+        public void onProgressChanged(SeekBar seekBar, int progress, boolean fromUser) {
+            //Convert from progress to a time stamp to display to the user
+            double videoDuration = mPlayerView.getDuration();
+            double currentTime = mPlayerView.getDuration() * ((double)progress / 100);
+            int finalMinutes = (int)videoDuration / 60;
+            int finalSeconds = (int)videoDuration % 60;
+            int currentMinutes = (int)currentTime / 60;
+            int currentSeconds = (int)currentTime % 60;
+            mSeekBarText.setText(
+                    padNumber(currentMinutes) +
+                    ":" +
+                    padNumber(currentSeconds) +
+                    " / " +
+                    padNumber(finalMinutes) +
+                    ":" +
+                    padNumber(finalSeconds));
+
+            //If the user moved the scrubber jump to that timestamp in the video
+            //Convert that to seconds into the video and seek that far
+            if(fromUser){
+                //progress is a value between 1-100 for percentage of video
+                double secondsToJumpTo = videoDuration * (progress / 100.0);
+                //jump that deep in seconds into the video
+                mPlayerView.seek(secondsToJumpTo);
+            }
+        }
+
+        //Helper function to help format the numbers in the time bar
+        private String padNumber(int number){
```

Comment:
You can accomplish this with `String.format()` instead.

Reply:
It's been updated to use string.format()